core/src/scss/components/card/_card--minimal.scss

```diff
@@ -0,0 +1,20 @@
+.su-card--minimal {
+  @include box-shadow($depth: 'flat', $stroke: 'none');
+  background-color: inherit;
+
+  .su-card__contents {
+    // override modular spacing at all breakpoints
+    @include grid-media-only(all) {
+      padding-left: 0;
+      padding-bottom: 0;
+      padding-right: 0;
+    }
+  }
+
+  &.su-card--horizontal {
+    // override modular spacing at all breakpoints
+    @include grid-media-only(all) {
+      padding: 0;
```

Comment:
And same here
`@include padding(0);`
I believe we're using these mixins for consistency for everything in Decanter.

Reply:
I actually prefer just specifying the padding directly. IMO the mixin saves very little typing and makes it slightly harder to read.

Reply:
> I actually prefer just specifying the padding directly. IMO the mixin saves very little typing and makes it slightly harder to read.

That might be a question for @sherakama ?  I don't particularly mind either way myself, but I believe at one point we agree that we'll use these mixins (padding and margin) wherever we can in Decanter 🤔

Reply:
l changed it to use the mixin, and I left-aligned the button.